### DISCUSSION TITLE
请教一个关于组件schema配置项默认值的问题

### DISCUSSION THREAD

**Discussion by cyt68** [alibaba/formily#2319] [Q&A 2.x] [answered]
相关库：alibaba/designable

请问组件schema配置项中的默认值只是用来适配显示的吗？是不是必须要和组件库一致呢？
比如说antd中的upload组件
![image](https://user-images.githubusercontent.com/20410138/137709387-82030196-211d-478c-9ae2-61094a797f8d.png)
![image](https://user-images.githubusercontent.com/20410138/137709529-c555d13b-5f6a-4be8-b4e5-4f17884bf663.png)
schema中默认值和组件的默认值都是'text'，那如果我想在这里把默认值改为'picture'应该怎么实现呢？
尝试了别的属性在schema里直接修改是没用的，编辑器显示会跟着变，但是传到组件的参数还是原来的默认值。

希望能指点迷津，感谢

**Comment by cning112** [comment, marked as answer]
直接使用`default: "picture"`, see https://react.formilyjs.org/api/shared/schema

**Reply by cyt68** [reply]
我给写到了createResource下的x-component-props能实现效果，不过你说这个应该才是正确做法👍
感谢
